os/pm/pm_procfs.c

```diff
@@ -447,43 +460,119 @@ static size_t power_devices_read(FAR struct file *filep, FAR char *buffer, size_
 }
 
 /****************************************************************************
- * Name: power_lock_write
+* 	Name: power_lock_write
+*   
+* 	Description:
+*	Lock a desired state by increasing the counter of the state.
+*
+* 	Input Parameters:
+*   filep             - File path of the power domain
+*   buffer            - Desired state to be unlocked
+*   buflen (NOT USED) - Length of the buffer
+*
+* 	Returned Value:
+*   0 (OK) means the state lock was successful.
+*	1 (Failed) means the input state was incorrect.
  ****************************************************************************/
 static size_t power_lock_write(FAR struct file *filep, FAR const char *buffer, size_t buflen)
 {
 	int state = atoi(buffer);
+	(void)buflen;
```

Comment:
Do we still need this if were not returning buflen anymore?

Reply:
Since the input argument is not used, it should be voided to prevent from unused variable warning.
The function definition follows the structure to all of the syscall API, lets keep them coherent.